ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
The Debian Python Policy requires a package to depend on `python3 (>= 3.Y)` (with `Y=5` in our case) (see https://www.debian.org/doc/packaging-manuals/python-policy/ch-module_packages.html#s-dependencies). So if we *would* require every Python package to declare that dependency it would for sure need to specify the correct version range.